data-prepper-plugins/log-http-source/README.md

```diff
@@ -0,0 +1,28 @@
+# Log HTTP Source
+
+This is a source plugin that supports HTTP protocol. Currently ONLY support Json UTF-8 codec for incoming request.
```

Comment:
Its more specific than just JSON UTF-8 right? This plugin only supports JSON arrays for the body of the incoming requests.

Reply:
So for pure json e.g. {"a": "b"}, it is called `json_lines` in fluentbit format and logstash codec plugin, I will add example in-line to clarify what json codec means